tests/ui/search.js

```diff
@@ -186,7 +186,7 @@ casper.test.begin('Test search potato', {
 
 casper.test.begin('Test search results header desktop', {
     test: function(test) {
-        helpers.startCasper({path: '/search?q=abc', viewport: 'tablet'});
+        helpers.startCasper({path: '/search?q=abc', viewport: 'desktop'});
```

Comment:
This header changes at the tablet width, I don't think this should change. If the test is failing then it's identifying a problem with the code not the test.